src/pages/watch/[videoId].tsx

```diff
@@ -104,10 +96,16 @@ export default function Watch({ video }: Video) {
                         overflow: 'auto',
                     }}
                 >
-                    <Toolbar/>
+                    <Toolbar />
                     <Container maxWidth="lg" sx={{ mt: 4, mb: 4 }}>
-                        <h1>{video.title}</h1>
-                        <video controls ref={videoRef} className="video-js" />
+                        <div style={{
+                            display: 'flex',
+                            alignItems: 'center',
+                            flexDirection: 'column'
+                        }}>
+                            <h1>{video.title}</h1>
+                            <video controls ref={videoRef} className="video-js" height={'500'} />
+                        </div>
```

Comment:
You could use the proper [Box](https://mui.com/material-ui/react-box/) and [Typography](https://mui.com/material-ui/react-typography/) components here